troubleshooting/known_issues/cray_console_node_in_CLBO_state.md

```diff
@@ -0,0 +1,85 @@
+# `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state after fresh CSM install
+
+## Issue Description
+
+After installing CSM 1.6, `CRAY-CONSOLE-NODE` pods intermittently goes into CLBO state. This happens because of
+permission issue with /var/log/conman directory inside the pods/container.
+
+## Error Identification
+
+When the issue occurs, the `CRAY-CONSOLE-NODE` pods will be in CLBO state. The logs from the pod will be throwing errors as follows:
+
+```text
+2023/02/03 16:30:37 Starting a new instance of command
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s7b0n0
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s21b3n0
+2023/02/03 16:30:37 Starting command process
+2023/02/03 16:30:37 Starting to parse file: /var/log/conman/console.x3000c0s9b0n0
+panic: runtime error: invalid memory address or nil pointer dereference
+[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x701979]
+
+goroutine 57 [running]:
+main.watchConsoleLogFile({0x835878, 0xc0003185c0}, {0xc000040920, 0xd})
+    /usr/local/golib/src/console_node/logAggregation.go:154 +0x2b9
+created by main.aggregateFile
+    /usr/local/golib/src/console_node/logAggregation.go:68 +0x14b
+```
+
+## Fix Description
+
+The workaround is to assign the correct permissions to the /var/log/conman directory inside the pods/container.
```

Comment:
```suggestion
The workaround is to assign the correct permissions to the `/var/log/conman` directory inside the container. This only needs to be done for a single pod, since the directory is shared between them.
```